src/helm/benchmark/annotation/harm_bench_annotator.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any
+
+from helm.benchmark.adaptation.request_state import RequestState
+from helm.benchmark.annotation.model_as_judge_annotator import ModelAsJudgeAnnotator
+
+PROMPT_TEMPLATE = """Here is an interaction between an adversarial user and a model,
```

Comment:
Prompt template has unnecessary newlines that will be sent to the annotator model.

The easiest way to deal with this is to simply remove the newlines, and then add `# noqa: E501` at the end of the closing triple quotes.